components/soap/src/main/java/org/switchyard/component/soap/composer/SOAPContextMapper.java

```diff
@@ -47,6 +51,12 @@
  */
 public class SOAPContextMapper extends BaseRegexContextMapper<SOAPBindingData> {
```

Comment:
This error is only happening on the SOAP Binding. That s why I included it here.